src/components/AnalysisPage/DisplayStatistic.jsx

```diff
@@ -2,15 +2,12 @@ import { Statistic, Tooltip } from "antd";
 import { InfoCircleOutlined } from "@ant-design/icons";
 import PropTypes from "prop-types";
 
-const GeoStatistic = ({ title, value }) => {
+const DisplayStatistic = ({ title, value }) => {
   const displayValue = (value) => {
     if (value === null) {
       return "--";
     }
 
-    // I put Shells and Boundary Edges back into GeoStatistic to keep the "--" display consistent
-    // It's also a plus that if we need a precise value for these two in the future,
-    // We're already set up for it
     if (title === "# of Shells" || title === "# of Boundary Edges") {
       return value;
     }
```

Comment:
I don't love that we have to specify the specific fields that are non-decimal because we'll have to keep this if statement always up to date as we add new fields to display. I'm not quite sure how to make it better though. Maybe splitting out to a separate displayStatistic for non-decimal values. Im not sure just thinking out loud

Reply:
you already saw the solution to all of these that you made comments on. but this is also taken care of hahah